cmd/agent/checkers.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package main
 
 import (
+	log "github.com/sirupsen/logrus"
```

Comment:
Nitpick: According to the [Go style guide](https://github.com/golang/go/wiki/CodeReviewComments) we follow this import should belong to the 3rd group.

Reply:
fixed.